web/wp-content/plugins/tmbi-adobe-dtm/legacy.php

```diff
@@ -0,0 +1,194 @@
+<?php
```

Comment:
Note that the files included here are meant to be moved to the appropriate places (e.g. plugins that provide the functionality and integrate with ADTM, or the theme).